src/drafts/SelectPanel2/SelectPanel.stories.tsx

```diff
@@ -471,6 +472,149 @@ export const TODO3NoCustomisation = () => {
   )
 }
 
+export const TODO4WithFilterButtons = () => {
+  const [filteredLabels, setFilteredLabels] = React.useState(data.branches)
+
+  const initialSelectedLabels: string[] = ['main']
+
+  // TODO: Single selection doesn't need an array
+  const [selectedLabelIds, setSelectedLabelIds] = React.useState<string[]>(initialSelectedLabels)
```

Comment:
note that the way the story is implemented, selecting a tag will also select a branch

https://github.com/primer/react/assets/39992/3a0d7069-6e40-425f-8893-bead002b2ab0

Reply:
Yes, I was only focusing on the tabs bit! I can clean this story up in a follow up PR?